File: messaging/services/service-create/service-create.5.x.rb
# Get twilio-ruby from twilio.com/docs/libraries/ruby
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

@service = @client.messaging.v1
                  .services
                  .create(
                    friendly_name: 'My First Service',
                    status_callback: 'http://requestb.in/1234abcd'
                  )

puts @service.sid

File: messaging/services/service-create/service-create.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Messaging.V1;


class Example
{
    static void Main (string[] args)
    {
      // Find your Account SID and Auth Token at twilio.com/console
      const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
      const string authToken = "your_auth_token";
      const string serviceFriendlyName = "My First Service";

      TwilioClient.Init(accountSid, authToken);

      var service = ServiceResource.Create(serviceFriendlyName);

      Console.WriteLine(service.Sid);
    }
}
